mythril/analysis/templates/report_as_text.jinja2

```diff
@@ -18,11 +18,11 @@ In file: {{ issue.filename }}:{{ issue.lineno }}
 
 --------------------
 {% endif %}
-{% if verbose and issue.debug %}
+{% if verbose and issue.tx_sequence %}
 --------------------
 Transaction Sequence:
 
-{{ issue.debug }}
+{{ issue.tx_sequence }}
```

Comment:
Shouldn't this be something like:
```suggestion
{{ issue.tx_sequence_users }}
```

Reply:
jinja takesin the issue.as_dict(), where this is tx_sequence